### ISSUE TITLE
Disabling TownyFlight v1.8.1

### ISSUE THREAD

**Issue by Chypik** [TownyAdvanced/TownyFlight#58]
Paper-132 (MC: 1.18.1)
Towny v0.97.5.8

```

[TownyFlight] Enabling TownyFlight v1.8.1
[15:00:33] [Server thread/ERROR]: Error occurred while enabling TownyFlight v1.8.1 (Is it up to date?)
java.lang.IllegalStateException: Queue full
	at java.util.AbstractQueue.add(AbstractQueue.java:98) ~[?:?]
	at java.util.concurrent.ArrayBlockingQueue.add(ArrayBlockingQueue.java:329) ~[?:?]
	at org.yaml.snakeyaml.emitter.Emitter.emit(Emitter.java:252) ~[snakeyaml-1.30.jar:?]
	at org.yaml.snakeyaml.serializer.Serializer.serializeComments(Serializer.java:224) ~[snakeyaml-1.30.jar:?]
	at org.yaml.snakeyaml.serializer.Serializer.serializeNode(Serializer.java:168) ~[snakeyaml-1.30.jar:?]
	at org.yaml.snakeyaml.serializer.Serializer.serializeNode(Serializer.java:206) ~[snakeyaml-1.30.jar:?]
	at org.yaml.snakeyaml.serializer.Serializer.serialize(Serializer.java:115) ~[snakeyaml-1.30.jar:?]
	at org.yaml.snakeyaml.Yaml.serialize(Yaml.java:366) ~[snakeyaml-1.30.jar:?]
	at org.bukkit.configuration.file.YamlConfiguration.saveToString(YamlConfiguration.java:89) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
	at com.gmail.llmdlio.townyflight.config.CommentedYamlConfiguration.save(CommentedYamlConfiguration.java:43) ~[TownyFlight-1.8.1.jar:?]
	at com.gmail.llmdlio.townyflight.config.TownyFlightConfig.loadConfig(TownyFlightConfig.java:107) ~[TownyFlight-1.8.1.jar:?]
	at com.gmail.llmdlio.townyflight.config.TownyFlightConfig.reload(TownyFlightConfig.java:18) ~[TownyFlight-1.8.1.jar:?]
	at com.gmail.llmdlio.townyflight.TownyFlight.loadConfig(TownyFlight.java:74) ~[TownyFlight-1.8.1.jar:?]
	at com.gmail.llmdlio.townyflight.TownyFlight.loadSettings(TownyFlight.java:68) ~[TownyFlight-1.8.1.jar:?]
	at com.gmail.llmdlio.townyflight.TownyFlight.onEnable(TownyFlight.java:32) ~[TownyFlight-1.8.1.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_18_R1.CraftServer.enablePlugin(CraftServer.java:564) ~[paper-1.18.1.jar:git-Paper-132]
	at org.bukkit.craftbukkit.v1_18_R1.CraftServer.enablePlugins(CraftServer.java:478) ~[paper-1.18.1.jar:git-Paper-132]
	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:727) ~[paper-1.18.1.jar:git-Paper-132]
	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:503) ~[paper-1.18.1.jar:git-Paper-132]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:313) ~[paper-1.18.1.jar:git-Paper-132]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1202) ~[paper-1.18.1.jar:git-Paper-132]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317) ~[paper-1.18.1.jar:git-Paper-132]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
[15:00:33] [Server thread/INFO]: [TownyFlight] Disabling TownyFlight v1.8.1
```
after reboot, just stopped working. repeated reboot did not give the result

**Comment by Chypik** [comment]
the config file was dropped and a new plugin generated. it all worked. what was the reason for the refusal?

**Comment by LlmDl** [comment]
This was fixed in TownyFlight 1.8.2.
